### DISCUSSION TITLE
Test cases medium 2

### DISCUSSION THREAD

**Discussion by Raxtesh** [IIC2133-PUC/2024-2#142] [Tarea 2] [answered]
Hola, tengo una consulta, porque en el output de los test cases en el evento "IN-CIRCLE", aparecen solamente en algunos los valores de x e y, pero no el radio? específicamente en las líneas 55, 82 y algunas otras mas.

**Comment by sebacarrascop** [comment, marked as answer]
hola!, efectivamente había un error, gracias por avisarlo, ya los arreglamos en el [drive](https://drive.google.com/drive/u/1/folders/15yEomUIX8zXnWa3_ohZSyKrMeLSup0Rz).
